backend/spotihue/spotihue.py

```diff
@@ -80,26 +91,60 @@ def _initialize_spotify(
         ) if redis_client else cache_handler.CacheFileHandler(cache_path='data/.spotify_token_cache')
 
         oauth_manager = oauth.SpotihueOauth(
-            client_id=spotify_client_id,
-            client_secret=spotify_client_secret,
-            redirect_uri=spotify_redirect_uri,
-            scope=spotify_scope,
-            open_browser=True,
+            client_id=client_id,
+            client_secret=client_secret,
+            redirect_uri=redirect_uri,
+            scope=scope,
+            open_browser=False,
             cache_handler=oauth_cache_handler
         )
         return spotipy.Spotify(auth_manager=oauth_manager)
 
-    def _initialize_hue(self, hue_bridge_ip_address: str) -> hue.HueBridge:
+    @staticmethod
```

Comment:
Why do we want this as a staticmethod? Just for learning purposes on my end

Reply:
By default, Python objects pass themselves into their methods/instance methods, but if a method doesn't need to use `self` at all, you can remove it as a param by making the method static (as you know). 
I think @staticmethod is sort of pointless in terms of functionality (1. we don't care about the tiny bit of memory briefly saved on the stack by omitting this `self` within this function's scope, and 2. @classmethod methods and standalone functions are also a thing). 
But I like it in terms of readability - it lets you know that the method does not need to know or care at all about what class it's in, or whether it's being called from a class (`Spotihue._initialize_hue`) or from a class instance (`spotihue_object._initialize_hue`). It'll return the same output for the same input regardless of the state of the class and its objects. BUT it's still declared within the class, rather than out in the open, because it's very specific to that class and what that class needs to accomplish (presumably/hopefully - otherwise it shouldn't be there).

Reply:
Thank you!